assets/lib/_variables.scss

```diff
@@ -185,6 +185,15 @@ $color-email-hover: #e8ae28 !default;
 $color-youtube: #cd201f !default;
 $color-youtube-hover: #B61B1B !default;
 
+// Focus
+$color-focus-BG: $color-yellow-dark !default;
+$color-focus-FG: $color-true-black !default;
```

Comment:
Could we make these all lower-case? That would be more consistent with other variables.